topics/reference_guide/work_with_icons_and_images.md

```diff
@@ -260,11 +260,47 @@ In this sample, the icon root folder is named <path>icons</path>:
 }
 ```
 
-If one new icon replaces several old icons, use JSON list format. Example from [`PlatformIconMappings.json`](%gh-ic%/platform/icons/src/PlatformIconMappings.json):
+If one new icon replaces several old icons, use a JSON list.
+Example from [`PlatformIconMappings.json`](%gh-ic%/platform/icons/src/PlatformIconMappings.json):
 
 ```json
 "vcs.svg": [
   "toolwindows/toolWindowChanges.svg",
   "vcs/branch.svg"
 ]
 ```
+
+### New UI Toolwindow Icons
+
+The New UI uses _outlined_ icons for tool windows that have a size of 20x20 pixel and 16x16 pixel in
+[compact mode](https://www.jetbrains.com/help/idea/new-ui.html#compact-mode).
+Plugin developers who want to provide all necessary variants of their tool window icons use the following
+naming scheme, where the first two files are icons with a size of 16x16 pixel.
+
+```shell
```

Comment:
We shouldn't use markup for visual purposes. Here we don't provide a Shell script but a simple list. We should use semantically correct markup.

I would consider using a simple bullet list for this or removing "shell".

Reply:
Fair enough. I wasn't sure either, but I wanted to mention the names explicitly.